.ci/openvino-onnx/Dockerfile

```diff
@@ -76,9 +76,4 @@ RUN make -j $(nproc) install
 
 # Run tests via tox
 WORKDIR /openvino/ngraph/python
-ENV NGRAPH_CPP_BUILD_PATH=/openvino/dist
-ENV LD_LIBRARY_PATH=/openvino/dist/lib
-ENV NGRAPH_ONNX_IMPORT_ENABLE=TRUE
-ENV PYTHONPATH=/openvino/bin/intel64/Release/lib/python_api/python3.8:${PYTHONPATH}
 RUN git clone --recursive https://github.com/pybind/pybind11.git
-RUN tox
```

Comment:
Please add a default `CMD` or `ENTRYPOINT`, which will run independently of `Jenskinsfile`